diff --git a/node_modules/@testing-library/dom/node_modules/pretty-format/build/index.d.ts b/node_modules/@testing-library/dom/node_modules/pretty-format/build/index.d.ts
index 8b3bc9e..5ccab32 100644
--- a/node_modules/@testing-library/dom/node_modules/pretty-format/build/index.d.ts
+++ b/node_modules/@testing-library/dom/node_modules/pretty-format/build/index.d.ts
@@ -4,7 +4,7 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import type * as PrettyFormat from './types';
+import * as PrettyFormat from './types';
 /**
  * Returns a presentation string of your `val` object
  * @param val any potential JavaScript object
